pages/advanced/api-safe-transaction-service/guides/api.md

```diff
@@ -0,0 +1,625 @@
+
+async function createTransactionAndSignByOwners() {
+  // Instantiate an EthAdapter for Owner A
+  const ethProvider = new ethers.JsonRpcProvider(config.RPC_URL)
+
+  const ownerA = new ethers.Wallet(config.OWNER_A_PRIVATE_KEY, ethProvider)
+  const ethAdapterOwnerA = new EthersAdapter({
+    ethers,
+    signerOrProvider: ownerA
+  })
+
+  // Initialize the Protocol Kit for Owner A
+  const protocolKitOwnerA = await Safe.create({
+    ethAdapter: ethAdapterOwnerA,
+    safeAddress: config.SAFE_ADDRESS
+  })
+
+  // Create Tx
+  const safeTransactionData: MetaTransactionData = {
+    to: config.TO,
+    value: config.VALUE,
+    data: '0x',
+    operation: OperationType.Call
+  }
+
+  const safeTransaction = await protocolKitOwnerA.createTransaction({
+    transactions: [safeTransactionData]
+  })
+
+  console.log('Tx created: %O', safeTransaction)
+
+  // Sign Tx by Owner A
+  const safeTxHash = await protocolKitOwnerA.getTransactionHash(safeTransaction)
+  const signatureOwnerA = await protocolKitOwnerA.signHash(safeTxHash)
+
+  console.log('Tx signed by Owner A: %s', signatureOwnerA.data)
+
+  // Initialize the Api Kit
+  const apiKit = new SafeApiKit({
+    chainId: 11155111n
+  })
+
+  // Create a Tx in Tx service with Signature of Owner A
+  const senderAddress = await ownerA.getAddress()
+
+  await apiKit.proposeTransaction({
+    safeAddress: config.SAFE_ADDRESS,
+    safeTransactionData: safeTransaction.data,
+    safeTxHash,
+    senderAddress,
+    senderSignature: signatureOwnerA.data
+  })
+
+  console.log('Tx proposed: %s', safeTxHash)
+
+  // Instantiate an EthAdapter for Owner B
+  const ownerB = new ethers.Wallet(config.OWNER_B_PRIVATE_KEY, ethProvider)
+  const ethAdapterOwnerB = new EthersAdapter({
+    ethers,
+    signerOrProvider: ownerB
+  })
+
+  // Initialize the Protocol Kit for Owner B
+  const protocolKitOwnerB = await Safe.create({
+    ethAdapter: ethAdapterOwnerB,
+    safeAddress: config.SAFE_ADDRESS
+  })
+
+  // Sign Tx by Owner B
+  const signatureOwnerB = await protocolKitOwnerB.signHash(safeTxHash)
+
+  console.log('Tx signed by Owner B: %s', signatureOwnerB.data)
+
+  // Add signature of owner B in Tx service
+  await apiKit.confirmTransaction(safeTxHash, signatureOwnerB.data)
+
+  console.log('Tx Owner B signature added in Tx service.')
+}
+```
+
+It is also possible to group multiple transactions into a single transaction and execute them in batch mode. To do so, just define the transactions as follows:
+
+```typescript
+// Create Tx
+const safeTransactionData: MetaTransactionData = {
+  to: config.TO,
+  value: config.VALUE,
+  data: '0x',
+  operation: OperationType.Call
+}
+
+const safeTransactionBData: MetaTransactionData = {
+  to: config.TO_B,
+  value: config.VALUE,
+  data: '0x',
+  operation: OperationType.Call
+}
+
+const safeTransaction = await protocolKitOwnerA.createTransaction({
+  transactions: [safeTransactionData, safeTransactionBData]
+})
+```
+
+#### Using safe-eth-py
```

Comment:
I think this documentation would be repeated. I think that would be repeated this documentation https://docs.safe.global/advanced/api-service-architecture/safe-transaction-service#offchain-messages.

Reply:
Yes, since the idea is to use the `Guides` section, it would be a good idea to add this section to the guides in another issue.

Reply:
Added an issue to review this page after add this new page `Guides` https://github.com/safe-global/safe-transaction-service/issues/1985